mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -422,199 +412,6 @@ def test_brainvision_vectorized_data():
     assert_array_almost_equal(raw._data[:, :2], first_two_samples_all_chs)
 
 
-def test_events():
-    """Test reading and modifying events."""
-    tempdir = _TempDir()
-    # Note: BrainVision event offsets are 1-based, mne offsets are 0-based.
-    # So in all tests below, the "onset" is 1 less than what's in the file
-
-    # check that events are read and stim channel is synthesized correctly
-    raw = read_raw_brainvision_old(vhdr_path, eog=eog, event_id=event_id)
-    events = raw._get_brainvision_events()
-    events = events[events[:, 2] != event_id['Sync On']]
-    assert_array_equal(events, [[486, 0, 253],
-                                [496, 0, 255],
-                                [1769, 0, 254],
-                                [1779, 0, 255],
-                                [3252, 0, 254],
-                                [3262, 0, 255],
-                                [4935, 0, 253],
-                                [4945, 0, 255],
-                                [5999, 0, 255],
-                                [6619, 0, 254],
-                                [6629, 0, 255],
-                                [7699, 0, 1]])
-
-    # check that events are read and stim channel is synthesized correctly and
-    # response triggers are shifted using the deprecated response_trig_shift.
-    with pytest.deprecated_call():
-        raw = read_raw_brainvision_old(vhdr_path, eog=eog,
-                                       response_trig_shift=1000,
-                                       event_id=event_id)
-    events = raw._get_brainvision_events()
-    assert_array_equal(events, [[486, 0, 253],
-                                [496, 0, 255],
-                                [1769, 0, 254],
-                                [1779, 0, 255],
-                                [3252, 0, 254],
-                                [3262, 0, 255],
-                                [4935, 0, 253],
-                                [4945, 0, 255],
-                                [5999, 0, 1255],
-                                [6619, 0, 254],
-                                [6629, 0, 255],
-                                [7629, 0, 5],
-                                [7699, 0, 1]])
-
-    # check that trig_shift_by_type works as well
-    raw = read_raw_brainvision_old(vhdr_path, eog=eog,
-                                   trig_shift_by_type={'response': 1000,
```

Comment:
this is no longer tested

Reply:
I mean `mne.io.brainvision.brainvision._event_id_func` and I don't think we can get rid of this `_event_id_func` because we need it for `read_annotation` IIRC.

Reply:
Does `_event_id_func` in https://codecov.io/gh/mne-tools/mne-python/pull/5733/src/mne/io/brainvision/brainvision.py render like this for everyone?

![image](https://user-images.githubusercontent.com/7044835/48844406-0a65f280-ed9a-11e8-9738-2a53938486ed.png)
￼
does this green/red thing mean that before it was green and now is red ??

Reply:
that is exactly it https://docs.codecov.io/docs/viewing-source-code#section-line-coverage-changes

Reply:
So if I understand properly, our CI coverage checks only the overall.
What I mean is that codecov was green with the following message `Absolute coverage decreased by -0.04% but relative coverage increased by +7.19% compared to d7c8e86`
While to me codecov should scream at us saying: whatever this PR is doing, is uncovering some lines that were previously cover

Reply:
Basically `_event_id_func` is only ever used by now-deprecated/removed functionality in `read_raw_brainvision`. It maybe should also be used in the `annotations`-based functionality, but come to think of it I think we decided to get rid of the magical function stuff and have people do it themselves and pass a complete `event_id`. So actually this seems okay.

Reply:
agreed is all dead-code. I pushed.